tests/localnet/test_localnet_start.py

```diff
@@ -168,11 +169,19 @@ def test_localnet_start_with_unparseable_docker_compose_version(app_dir_mock: Ap
     verify(result.output.replace(str(app_dir_mock.app_config_dir), "{app_config}").replace("\\", "/"))
 
 
-@pytest.mark.usefixtures("health_success")
+@pytest.mark.usefixtures("health_success", "_localnet_up_to_date")
 def test_localnet_start_with_gitpod_docker_compose_version(app_dir_mock: AppDirs, proc_mock: ProcMock) -> None:
     proc_mock.set_output("docker compose version --format json", [json.dumps({"version": "v2.10.0-gitpod.0"})])
 
     result = invoke("localnet start")
 
     assert result.exit_code == 0
     verify(result.output.replace(str(app_dir_mock.app_config_dir), "{app_config}").replace("\\", "/"))
+
+
+@pytest.mark.usefixtures("proc_mock", "health_success", "_localnet_out_of_date")
+def test_localnet_start_out_date(app_dir_mock: AppDirs) -> None:
```

Comment:
Might be useful to add few more tests testing edge cases when you have local docker inspect invocation failing on some exception (should be handled without raising any unhandled exceptions). Similarly testing a scenario where remote local inspect throws an error, as expected though it should print a debug warning but user of the cli shouldn't see any errors/logs and instead its just continues with the execution of the command.